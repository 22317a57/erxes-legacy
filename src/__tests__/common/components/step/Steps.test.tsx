import { mount, shallow } from 'enzyme';
import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Steps from '../../../../modules/common/components/step/Steps';

describe('Group component', () => {
  const defaultProps = {
    children: [],
    active: 0
  };

  test('renders Steps successfully', () => {
    shallow(<Steps {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<Steps {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });

  test('snapshot matches', () => {
    const rendered = renderer.create(<Steps {...defaultProps} />).toJSON();

    expect(rendered).toMatchSnapshot();
  });
});
