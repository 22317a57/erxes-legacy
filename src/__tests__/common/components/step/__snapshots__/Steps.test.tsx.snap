// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group component snapshot matches 1`] = `
<div
  className="sc-bdVaJa fkrXdF"
/>
`;
